fe1-web/src/features/evoting/objects/Election.ts

```diff
@@ -127,8 +128,12 @@ export class Election {
     this.end = obj.end;
     this.questions = obj.questions;
     this.questionResult = obj.questionResult;
-    // Sets the election status automatically
-    this.electionStatus = Election.getElectionStatus(obj.start, obj.end);
+    // Sets the election status automatically if none is passed
+    if (obj.electionStatus === undefined) {
+      this.electionStatus = Election.getElectionStatus(obj.start, obj.end);
```

Comment:
Would it make sense to default to `NOT_RUNNING` as a more robust option ? 
We'd expect messages to trigger the change anyway, under the new functioning.

Reply:
Yes, definitely! This was also one of the things I wanted to change after the discussion here on GitHub. Before this was the only way the election status was determined and there was some really interesting code making it all work.

I actually removed this default state and required it to be always passed explicitly now as I feel like this is an even clearer option.